Assets/HoloToolkit/SpatialUnderstanding/Readme.md

```diff
@@ -0,0 +1,193 @@
+Copied from [Spatial Mapping In Unity](https://developer.microsoft.com/en-us/windows/holographic/spatial_mapping_in_unity) page of the Microsoft Dev Center Holographic Documentation.
```

Comment:
That's a valid point.  What is the frequency of updates to the documentation?

I simply did just that for the Wiki.

I think it would be beneficial to also include the link to the case study.